cli/src/cmd/forge/install.rs

```diff
@@ -168,6 +172,21 @@ fn install_as_submodule(
     Ok(())
 }
 
+// check that there are no modification in git working/staging area
+fn git_status_clean() -> eyre::Result<bool> {
+    let output = Command::new("git")
+        .args(&["status", "--short"])
+        .stdout(Stdio::piped())
+        .stderr(Stdio::piped())
+        .output()?;
+    if !&output.status.success() {
+        let stderr = String::from_utf8_lossy(&output.stderr);
```

Comment:
can we check if this is a config error, in which case we emit a warning but proceed?

Reply:
i can't locate integration tests failing due to this currently, either, not sure why

but try commenting out the `user` section in global `.gitconfig`, and running `cli` package tests locally will reveal tests that fail because of this issue, e.g.
```
cmd::can_clean_config
cmd::can_emit_extra_output
cmd::can_emit_multiple_extra_output
cmd::can_init_non_empty
cmd::can_init_quiet
cmd::can_init_repo_repeatedly_with_force
cmd::can_init_repo_with_config
cmd::can_init_vscode
config::can_detect_config_vals
config::can_detect_lib_foundry_toml
config::can_get_evm_opts
config::can_override_config
config::can_parse_dapp_libraries
config::can_prioritise_closer_lib_remappings
config::can_set_config_values
config::can_update_libs_section
test_cmd::can_test_repeatedly
```